docs/visualization/lupton_rgb.rst

```diff
@@ -0,0 +1,70 @@
+.. _astropy-visualization-rgb:
+
+*************************
+Creating color RGB images
+*************************
+
+`Lupton et al. (2004)`_ describe an "optimal" algorithm for producing red-green-
+blue composite images from three separate high-dynamic range arrays. This method
+is implemented in `~astropy.visualization.make_lupton_rgb` as a convenience
+wraper function and an associated set of classes to provide alternate scalings.
+The SDSS SkyServer color images were made using a variation on this technique.
+To generate a color PNG file with the default (arcsinh) scaling:
+
+.. _Lupton et al. (2004): http://adsabs.harvard.edu/abs/2004PASP..116..133L
+
+.. doctest-skip::
+
+>>> import numpy as np
+>>> from astropy.visualization import make_lupton_rgb
+>>> imageR = np.random.random((100,100))
+>>> imageG = np.random.random((100,100))
+>>> imageB = np.random.random((100,100))
+>>> image = make_lupton_rgb(image_r, image_g, image_b, filename='randoms.png')
```

Comment:
@parejkoj - the six lines above should be indented for them to be considered inside the ``doctest-skip`` directive. This is what was causing your documentation builds to fail.